common/Configuring_CloudForms.adoc

```diff
@@ -0,0 +1,155 @@
+[[Configuring-cloudforms]]
+== Configuring {product-title}
```

Comment:
...by 'top level' here, I mean the article title. Note also that there's no metadata here, but I think you might want to include some - see the first several lines of [this example](https://raw.githubusercontent.com/openshift/openshift-docs/master/architecture/infrastructure_components/kubernetes_infrastructure.adoc).